RecoEgamma/EgammaElectronProducers/python/ecalDrivenElectronSeeds_cfi.py

```diff
@@ -34,11 +34,12 @@
 from Configuration.Eras.Modifier_phase2_hgcal_cff import phase2_hgcal
 phase2_hgcal.toModify(
     ecalDrivenElectronSeeds,
-    endcapSuperClusters = cms.InputTag('particleFlowSuperClusterHGCal')
-)
-phase2_hgcal.toModify(
-    ecalDrivenElectronSeeds.SeedConfiguration,
-    allowHGCal = cms.bool(True)
+    endcapSuperClusters = cms.InputTag('particleFlowSuperClusterHGCal'),
+    SeedConfiguration = dict( allowHGCal = cms.bool(True) )
 )
 
 
+# create ecal driven seeds for electron using HGCal Multiclusters
+ecalDrivenElectronSeedsFromMultiCl = ecalDrivenElectronSeeds.clone(
+  endcapSuperClusters = cms.InputTag('particleFlowSuperClusterHGCalFromMultiCl'))
```

Comment:
rhs type specs are more dangerous than useful because they allow to make sure that the parameter actually exists. 
[OK to stay, considering that the rest of the file is not following this style and if nothing else more important pops up in this round of review]